go-controller/pkg/node/gateway_udn.go

```diff
@@ -107,11 +107,7 @@ func (b *bridgeConfiguration) getBridgePortConfigurations() ([]*bridgeUDNConfigu
 }
 
 // addNetworkBridgeConfig adds the patchport and ctMark value for the provided netInfo into the bridge configuration cache
-func (b *bridgeConfiguration) addNetworkBridgeConfig(
-	nInfo util.NetInfo,
-	nodeSubnets []*net.IPNet,
-	masqCTMark, pktMark uint,
-	v6MasqIPs, v4MasqIPs *udn.MasqueradeIPs) error {
+func (b *bridgeConfiguration) addNetworkBridgeConfig(nInfo util.NetInfo, nodeSubnets, mgmtIPs []*net.IPNet, masqCTMark, pktMark uint, v6MasqIPs, v4MasqIPs *udn.MasqueradeIPs) error {
```

Comment:
nit: I find this a lot harder to parse visually than before - I think one line per attribute removes a lot of burder from the code reader / reviewer.